modules/ROOT/partials/import-database.adoc

```diff
@@ -43,6 +43,8 @@ If the database you want to upload is running an earlier version of Neo4j, pleas
 ====
 The `neo4j-admin push-to-cloud` command in Neo4j 4.4 and earlier is not compatible with instances encrypted with xref:platform/security/encryption.adoc#_customer_managed_keys[Customer Managed Keys].
 Use `neo4j-admin database` upload in Neo4j 5 to upload data to instances encrypted with Customer Managed Keys.
+
+For Neo4j Version 4 Azure instances encrypted with Customer Managed Keys, use Neo4j Data Importer on the Import Tab of Workspace to load data, as `neo4j-admin database upload` is not supported. Please see the https://neo4j.com/docs/data-importer/current/introduction[Data Importer documentation].
```

Comment:
```suggestion
For Neo4j 4.x instances encrypted with Customer Managed Keys, use Neo4j Data Importer to load data, as `neo4j-admin database upload` is not supported. 
See the https://neo4j.com/docs/data-importer/current/introduction[Data Importer documentation] for more information.
```

Reply:
We don't mention anywhere that CMK for Neo4j 4.x is only available for Azure. Is that the case? Also, we try not to talk about Workspace.

Reply:
I updated the suggestion since it does only apply to Azure, but we should still try to leave out any mention of Workspace.

Reply:
i see why, workspace will be removed when upx is launched